client/app/assets/styles/variables.js

```diff
@@ -1,4 +1,14 @@
 /* eslint-env node */
+const fontSizeSmall = '13px';
+const fontSize = '14px';
+const fontSizeBig = '16px';
```

Comment:
There are also the base font sizes for mobile and desktop. These should probably be combined with those?